runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStyle.java

```diff
@@ -32,10 +32,13 @@
 	AGGRESSIVE("Aggressive", Skill.STRENGTH),
 	DEFENSIVE("Defensive", Skill.DEFENCE),
 	CONTROLLED("Controlled", Skill.ATTACK, Skill.STRENGTH, Skill.DEFENCE),
+	ACCURATE_RANGING("Accurate Ranging", Skill.RANGED),
 	RANGING("Ranging", Skill.RANGED),
 	LONGRANGE("Longrange", Skill.RANGED, Skill.DEFENCE),
 	CASTING("Casting", Skill.MAGIC),
 	DEFENSIVE_CASTING("Defensive Casting", Skill.MAGIC, Skill.DEFENCE),
+	AIM_AND_FIRE("Aim and Fire"),
+	BLOCK("Blocking"),
 	OTHER("Other");
```

Comment:
What is this used for?

Reply:
> What is this used for?

`AIM_AND_FIRE` is used by the Gun category, which is currently only used by Fixed Device

`BLOCK` is used by the Dinh's Bulwark

Reply:
This comment is on the `OTHER` line, which looks unused to me.

Reply:
Ah mis-observed, my bad. `OTHER` is what was used prior for those two mentioned unique combat styles. I believe it is being retained just in case there is another style not covered by the enum, but perhaps Spedwards can elaborate better.